### DISCUSSION TITLE
Plotting tracking history in real-time

### DISCUSSION THREAD

**Discussion by canliu0414** [bonsai-rx/bonsai#1855] [Q&A]
Hi,

I am trying to make a workflow that captures video from basler camera and do realtime tracking; I also want to visualize the tracking history, as maybe a simple plot of dots on blank background (overlaying on video is probably more computation heavy?). Could anyone give some suggestions on how to do the plotting step? thank you!

![Screenshot 2024-06-21 102308](https://github.com/bonsai-rx/bonsai/assets/49167749/b7d07a56-d009-49b6-8cd3-42b79184f478)

**Comment by bruno-f-cruz** [comment]
If it is just for visualization purposes, the simplest would be to use mashups:

![image](https://github.com/bonsai-rx/bonsai/assets/7049351/4568487c-204e-4757-bb7c-04d9bf786bd9)

If you left click on top of the visualizer of the last node, the history of the point will also be plotted. Alternatively, you could code something custom. [See here for an example.](https://groups.google.com/g/bonsai-users/c/CxsCtVjk2IE/m/YKTjtbCtGwAJ)

**Comment by canliu0414** [comment]
Hi,

Thanks for the reply. For visualization purpose, the visualizer mapping is working. Thank you very much!
